spec/metrics_spec.adoc

```diff
@@ -417,45 +417,11 @@ A server SHOULD implement TLS encryption by default
 
 === Required + Vendor specific metrics
 
-Q: Do we want to mandate this or can/should each vendor do as they like?
+Each vendor of a Microprofile-compliant server can implement this part in a vendor-specific way. The
+values can be backed into the server.
+The Appendix shows a possible data format.
```

Comment:
It's not clear from the above what it is that can be implemented in a vendor-specific way.  Is what is in the appendix something an end user could configure?  Would  that mean then that users are allowed to reduce the set of metrics that are included in the base/vendor set?